setup.py

```diff
@@ -32,6 +32,7 @@
         'cf_units',
         'cython',
         # 'scitools-iris',  # Only iris 2 is on PyPI
+        'jinja2',
```

Comment:
Also add this package to meta.yml for the conda build

Reply:
Done